.gitignore

```diff
@@ -11,3 +11,8 @@ upgrade
 html/
 doxygen/doxypy-0.4.2/
 .pytest_cache/
+
+bin/
```

Comment:
this drives me crazy, this should be in `.gitignore` already for a very long time.

Reply:
@driftx there are activation scripts for python env and similar


    $ ls bin/
    activate  activate.csh  activate.fish  activate.nu  activate.ps1  activate_this.py  ccm  pip  pip3  pip-3.10  pip3.10  python  
    python3  python3.10  wheel  wheel3  wheel-3.10  wheel3.10

Reply:
offline agreed we can keep it like it is here